openunreid/apis/runner.py

```diff
@@ -311,7 +314,7 @@ def save(self, mAP=None):
             self._best_mAP = max(self._best_mAP, mAP)
             print(
                 bcolors.OKGREEN
-                + "\n * Finished epoch {:3d}  mAP: {:5.1%}  best: {:5.1%}{}\n".format(
+                + "\n * Finished epoch {:3d}  current: {:5.1%}  best: {:5.1%}{}\n".format(
```

Comment:
"current" here is not so good, please specify whether the metric is mAP or CMC here.